src/core/MPIChannel.cpp

```diff
@@ -40,7 +40,7 @@
 #include "MPIChannel.h"
 
 #include "MPIContext.h"
-#include "MessageHeader.h"
```

Comment:
Oh, this include should already have been removed! There is a different MPIHeader.h included in MPIChannel.h